src/css/styles.css

```
@@ -0,0 +1,74 @@
+/* Reset some basic elements (optional) */
```

Automated Code Review Suggestion:

```suggestion
  overflow-y: auto;
```
Add overflow-y to the sidebar to prevent content from overflowing the sidebar and becoming inaccessible if it exceeds the viewport height.